jicofo/rootfs/defaults/jicofo.conf

```diff
@@ -7,7 +7,7 @@
 {{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "0" | toBool -}}
 {{ $ENABLE_RECORDING := .Env.ENABLE_RECORDING | default "0" | toBool -}}
 {{ $ENABLE_OCTO := .Env.ENABLE_OCTO | default "0" | toBool -}}
-{{ $ENABLE_OCTO_SCTP := .Env.ENABLE_OCTO_SCTP | default .Env.ENABLE_SCTP | toBool -}}
+{{ $ENABLE_OCTO_SCTP := .Env.ENABLE_OCTO_SCTP | default $ENABLE_SCTP | toBool -}}
```

Comment:
This would work, except that we need to use the toBool which only accepts a string. I had it this way to begin with but it fails

Reply:
```
{{ $ENABLE_SCTP_TMP := .Env.ENABLE_SCTP | default "0" }}
{{ $ENABLE_SCTP := $ENABLE_SCTP_TMP | toBool -}}
{{ $ENABLE_OCTO_SCTP := .Env.ENABLE_OCTO_SCTP | default $ENABLE_SCTP_TMP | toBool -}}
```
Not really elegant, but works.

Reply:
Hum, I suppose we could fix tpl to do it? https://github.com/jitsi/tpl/blob/0a1fc48a5bcb8cbd88db78bb59b8f76818058af4/func.go#L45